core/src/main/scala/cromwell/core/simpleton/WomValueBuilder.scala

```diff
@@ -15,16 +15,16 @@ import scala.language.postfixOps
 object WomValueBuilder {
 
   /**
-    * Looks for a WDL identifier possibly followed by a metacharacter and more stuff.
+    * Looks for a WOM identifier possibly followed by a metacharacter and more stuff.
     *
     * Capture groups:
     *
     * <ol>
-    * <li>A WDL identifier</li>
-    * <li>Possibly a metacharacter and more stuff after the WDL identifier</li>
+    * <li>A WOM identifier</li>
+    * <li>Possibly a metacharacter and more stuff after the WOM identifier</li>
     * </ol>
     */
-  private val IdentifierAndPathPattern = "^([a-zA-Z][a-zA-Z0-9_]*)(.*)".r
+  private val IdentifierAndPathPattern = "^([a-zA-Z][-a-zA-Z0-9_]*)(.*)".r
```

Comment:
Currently this expands the simpleton key character space to include dash characters. The use of a dash in 'ps-stdOut' was terminating identifier parsing prematurely and causing that file not to be populated the way downstream code expected.

But AFAICT there aren't any restrictions on CWL identifier naming at all so our code should either be more robust than this or at least generate a non-useless fail-fast error message if we find ourselves in this situation which is most definitely not what we're doing now.